rootfs/Dockerfile

```diff
@@ -61,7 +59,13 @@ RUN bash -xeu -c ' \
   for dir in "${writeDirs[@]}"; do \
     mkdir -p ${dir}; \
     chown -R www-data.www-data ${dir}; \
-  done'
+  done' \
+  # LD_LIBRARY_PATH does not work so below is needed for  opentelemetry/other modules
+  # Put libs of newer modules under `/modules_mount/<other>/lib` and add that path below
+  # Could get complicated arch specific paths become a need
+  && touch /etc/ld-musl-x86_64.path \
```

Comment:
we don't need `touch` it first, right?

Reply:
correct. I removed the touch command from both Dockerfiles